packages/events/examples/events_threading.rs

```diff
@@ -0,0 +1,72 @@
+//! Example demonstrating both thread-safe and single-threaded event variants.
```

Comment:
Please split this into separate examples, one for thread-safe and one for single-threaded events.